src/sagemaker/fw_utils.py

```diff
@@ -53,7 +53,14 @@
 )
 
 VALID_PY_VERSIONS = ["py2", "py3"]
-VALID_EIA_FRAMEWORKS = ["tensorflow", "tensorflow-serving", "mxnet", "mxnet-serving"]
+VALID_EIA_FRAMEWORKS = [
+    "tensorflow",
+    "tensorflow-serving",
+    "mxnet",
+    "mxnet-serving",
+    "pytorch",
```

Comment:
I don't recall exactly where this logic is used, but doesn't "pytorch" technically still not support EI since none of the "pytorch" images that also had the serving stack supported EI? it's just "pytorch-serving" with the EI support, right?

Reply:
That makes sense. The only version of pytorch that supports eia is 1.3.1 and that's after spliting. So only ``pytorch-serving`` is EIA valid.